lambda_uploader/uploader.py

```diff
@@ -171,3 +174,16 @@ def _validate_package_size(self, pkg):
         '''
         if path.getsize(pkg) > MAX_PACKAGE_SIZE:
             LOG.warning("Size of your deployment package is larger than 50MB!")
+
+    def _format_vpc_config(self):
+        '''
+        Returns {} if the VPC config is set to None by Config,
+        returns the formatted config otherwise
+        '''
+        if self._config.raw['vpc']:
```

Comment:
Yeah that is going to happen when the vpc is set to None. Hmm https://github.com/DeviaVir/lambda-uploader/blob/feature_vpc_config/lambda_uploader/config.py#L119-L124 should probably handle valid keys set to None.